variables.tf

```
@@ -1,6 +1,12 @@
 ########################################
 # General vars
 ########################################
+variable "create_secret" {
+  default     = true
+  description = "If true, this module does nothing (since tf doesn't support conditional modules)"
```

shouldn't it be if false? 
```suggestion
  description = "If false, this module does nothing (since tf doesn't support conditional modules)"
```

---

good catch.